### PR TITLE
Implement design refinements to the organiser AutoSuggest component

### PR DESCRIPTION
**Pull request by bp323**

### CODE REVIEW

**Comment by bp323** [comment]
Assigning to @Coenego for review.

File: shared/gh/js/views/gh.admin-batch-edit-organiser.js [hunk 1]
```diff
@@ -173,12 +179,54 @@ define(['gh.core', 'jquery-autosuggest'], function(gh) {
...
+        // Only close the input if the focus was lost on an element outside of the organiser container
+        if (!$(ev.target).closest('.gh-batch-event-organisers').length || (ev.relatedTarget && !$(ev.relatedTarget).closest('.gh-batch-event-organisers').length)) {
+            $('.gh-batch-event-organisers').addClass('gh-batch-event-collapsed');
+            // Update the lecture count in the placeholder
+            updateAutoSuggestPlaceholder();
+        }
+    };
+
+    /**
+     * Open the AutoSuggest component by removing the class from it's parent that hid all selected users from view
```
Comment (Coenego) [inline shared/gh/js/views/gh.admin-batch-edit-organiser.js]: hide

**Comment by Coenego** [comment]
Nice work! Only left a minor issue left. Reassigning to @bp323

**Comment by bp323** [comment]
Reassigning to @Coenego for review

**Pull request merged**